### ISSUE TITLE
Add to favorite action - favorites counter bumped but back to it's original state after page being refreshed

### ISSUE THREAD

**Issue by melezhik** [metacpan/metacpan-web#1805]
Hi!

Probably you know this, but how to reproduce:

1. get loggin to [metacpan](http://metacpan.org/)
2. found some module I want to add to favorites
3. click "add to favorite" button , counter gets bumped 
![metacpan-bump-counter](https://raw.githubusercontent.com/melezhik/screenshots/master/metacpan-bump-counter.png)
4. refresh the page - counter gets back to it's original state 
![metacpan-bump-counter](https://raw.githubusercontent.com/melezhik/screenshots/master/metacpan-bump-counter-back.png)

**Comment by tsibley** [comment]
@melezhik Thanks for the report!  I'm having trouble reproducing this myself, but I have some suspicions as to what might be happening.  Would you try logging out of metacpan.org and then logging in again?  I suspect the issue is with sessions that were migrated.

**Comment by melezhik** [comment]
@tsibley I have tried to log out and log in, but this did not help. The issue is still here.

**Comment by p-alik** [comment]
I observed the same counter behaviour but the module I wished appears in my favourites

**Comment by ranguard** [comment]
I wonder if it's a caching/timing issue

**Comment by melezhik** [comment]
@ranguard , hi :-) ! nice to meet you ...

 I repeated my steps in anonymous firefox mode ( to get my cache off ) , but the issue is still here.

**Comment by ranguard** [comment]
@melezhik thanks - sorry I should have been clearer - I meant our CDN caching rather than browser caching

**Comment by melezhik** [comment]
ahhh, I see now : )

Btw, I have been running into this type of issue for months, sorry guys to let you know only now ...

**Comment by ranguard** [comment]
oh! - well we only just switched to more caching so that's really useful to know

**Comment by melezhik** [comment]
ok, good to know then ...

**Comment by KES777** [comment]
Any progress here? O:-)

**Comment by oalders** [comment]
This is *possibly* now fixed. Can anyone confirm/deny?

**Comment by melezhik** [comment]
Ohhh , 10 years passed 😊 I no longer use MetaCpan , sorry

**Comment by oalders** [comment]
Yeah, I mean I'm not particularly proud of these 10 year old tickets. 😅 I hope you're well, @melezhik!

**Comment by melezhik** [comment]
@oalders not a problem at all 😊, doing good , hopefully you do well as well

**Comment by oalders** [comment]
All good over here as well, @melezhik. I'm currently not working and my kids are teenagers now, so I finally found the time to sit down to try to fix some of these bugs. 😊

**Comment by melezhik** [comment]
oh, blessed times, I need to wait a bit longer for that , sigh )))
